src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerPropertyConverter.cs

```diff
@@ -122,7 +123,11 @@ private bool IsMultipleDataType(int dataTypeId, string propertyEditorAlias)
         /// </returns>
         public override bool IsConverter(PublishedPropertyType propertyType)
         {
-            return propertyType.PropertyEditorAlias.Equals(Constants.PropertyEditors.MediaPicker2Alias);
+            if (UmbracoConfig.For.UmbracoSettings().Content.EnablePropertyValueConverters)
```

Comment:
The reason this isn't here IIRC is because MediaPicker2Alias is a brand new property editor, one that wouldn't exist in previous versions so there would be no reason to disable the property converter for this since it's new and if people are using it it will just return what it is supposed to.